aspnet/conceptual-overview/aspnet.rst

```diff
@@ -1,37 +1,37 @@
 Introduction to ASP.NET Core
 ============================
 
-By `Daniel Roth`_
+By `Daniel Roth`_ and `Rick Anderson`_
 
 ASP.NET Core is a significant redesign of ASP.NET. This topic introduces the new concepts in ASP.NET Core and explains how they help you develop modern web apps.
 
 .. contents:: Sections:
   :local:
   :depth: 1
-  
+
 What is ASP.NET Core?
 ---------------------
 
-ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web applications using .NET. We built it from the ground up to provide an optimized development framework for apps that are either deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core applications cross-platform on Windows, Mac and Linux. ASP.NET Core is fully open source on `GitHub <https://github.com/aspnet/home>`_.
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web apps using .NET. We built it from the ground up to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at `GitHub <https://github.com/aspnet/home>`_.
 
 Why build ASP.NET Core?
 -----------------------
 
-The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web applications, and over the years we have added and evolved many, many capabilities to it.
+The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web apps, and over the years we have added and evolved many capabilities to it.
```

Comment:
There's that word web. Is using it limiting the perceived scope of ASP.NET Core? Is ASP.NET core really limited to web sites & apps?